src/applications/hca/selectors.js

```diff
@@ -25,7 +26,7 @@ export const noESRRecordFound = state =>
   selectEnrollmentStatus(state).noESRRecordFound;
 export const isShowingHCAReapplyContent = state =>
   selectEnrollmentStatus(state).showHCAReapplyContent;
-export const isInESR = state => {
+export const hasApplicationInESR = state => {
```

Comment:
The old name of this has bugged me for some time so and I'm finally changing it.